### PR TITLE
Update the readme and properties configuration of servicebus multibinders.

### PR DESCRIPTION
**Pull request by hui1110**
Modify the directory of servicebus queue multibinders.
Update the readme and properties configuration of servicebus multibinders.

### CODE REVIEW

File: servicebus/spring-cloud-azure-stream-binder-servicebus/servicebus-multibinders/src/main/resources/application-sp.yaml [hunk 1]
```diff
@@ -45,10 +42,16 @@ spring:
                     connection-string: [ servicebus-namespace-2-connection-string ]
```
Comment (moarychan) [inline servicebus/spring-cloud-azure-stream-binder-servicebus/servicebus-multibinders/src/main/resources/application-sp.yaml:42]: It's better to change to AZURE_XXX_SERVICE_CONNECTION_STRING.

**Review by moarychan — CHANGES_REQUESTED** [review]

File: servicebus/spring-cloud-azure-stream-binder-servicebus/servicebus-topic-binder/src/main/resources/application-sp.yaml [hunk 2]
```diff
@@ -2,10 +2,10 @@ spring:
...
       credential:
-        client-id: [ client-id ]
-        client-secret: [ client-secret ]
+        client-id: ${AZURE_CLIENT_ID}
+        client-secret: ${AZURE_CLIENT_SECRET}
       profile:
-        tenant-id: [ tenant-id ]
+        tenant-id: ${AZURE_TENANT_ID}
 
 #    Uncomment below configurations if you want to enable auto creating resources.
```
Comment (yiliuTo) [inline servicebus/spring-cloud-azure-stream-binder-servicebus/servicebus-topic-binder/src/main/resources/application-sp.yaml:10]: Please remove the configuration of auto-creating-resources in all the configuration files and readme.

File: servicebus/spring-cloud-azure-stream-binder-servicebus/servicebus-topic-binder/src/main/resources/application-mi.yaml [hunk 3]
```diff
@@ -2,9 +2,9 @@ spring:
   cloud:
     azure:
       credential:
-        managed-identity-client-id: [ managed-identity-client-id ]
+        managed-identity-client-id: ${AZURE_MANAGED_IDENTITY_CLIENT_ID]
```
Comment (yiliuTo) [inline servicebus/spring-cloud-azure-stream-binder-servicebus/servicebus-topic-binder/src/main/resources/application-mi.yaml:5]: change ${AZURE_MANAGED_IDENTITY_CLIENT_ID] to ${AZURE_MANAGED_IDENTITY_CLIENT_ID}, same for other places

**Review by yiliuTo — CHANGES_REQUESTED** [review]

File: servicebus/spring-cloud-azure-stream-binder-servicebus/servicebus-multibinders/pom.xml [hunk 4]
```diff
@@ -10,9 +10,9 @@
   </parent>
   <modelVersion>4.0.0</modelVersion>
 
-  <artifactId>spring-cloud-azure-sample-stream-binder-servicebus-multiple-queue</artifactId>
+  <artifactId>spring-cloud-azure-sample-stream-binder-servicebus-multiple</artifactId>
   <groupId>com.azure.spring</groupId>
-  <version>1.0.0</version> <!-- {x-version-update;com.azure.spring:spring-cloud-azure-sample-stream-binder-servicebus-multiple-queue;dependency} -->
+  <version>1.0.0</version> <!-- {com.azure.spring:spring-cloud-azure-sample-stream-binder-servicebus-multiple;dependency} -->
```
Comment (yiliuTo) [inline servicebus/spring-cloud-azure-stream-binder-servicebus/servicebus-multibinders/pom.xml:15]: you can just delete the whole lable: . 
Same for other pom.xml.

File: servicebus/spring-cloud-azure-stream-binder-servicebus/servicebus-multibinders/pom.xml [hunk 5]
```diff
@@ -50,7 +50,7 @@
       <plugin>
         <groupId>com.microsoft.azure</groupId>
         <artifactId>azure-webapp-maven-plugin</artifactId>
-        <version>1.14.0</version> <!-- {x-version-update;com.microsoft.azure:azure-webapp-maven-plugin;external_dependency} -->
+        <version>1.14.0</version> <!-- {com.microsoft.azure:azure-webapp-maven-plugin;external_dependency} -->
```
Comment (yiliuTo) [inline servicebus/spring-cloud-azure-stream-binder-servicebus/servicebus-multibinders/pom.xml:53]: please help to delete the whole <build> part.

File: servicebus/spring-cloud-azure-stream-binder-servicebus/servicebus-queue-binder/pom.xml [hunk 6]
```diff
@@ -52,7 +52,7 @@
       <plugin>
         <groupId>com.microsoft.azure</groupId>
         <artifactId>azure-webapp-maven-plugin</artifactId>
-        <version>1.14.0</version> <!-- {x-version-update;com.microsoft.azure:azure-webapp-maven-plugin;external_dependency} -->
+        <version>1.14.0</version> <!-- {com.microsoft.azure:azure-webapp-maven-plugin;external_dependency} -->
```
Comment (yiliuTo) [inline servicebus/spring-cloud-azure-stream-binder-servicebus/servicebus-queue-binder/pom.xml:55]: same here

File: servicebus/spring-cloud-azure-stream-binder-servicebus/servicebus-queue-binder/pom.xml [hunk 7]
```diff
@@ -12,7 +12,7 @@
 
   <artifactId>spring-cloud-azure-sample-stream-binder-servicebus-queue</artifactId>
   <groupId>com.azure.spring</groupId>
-  <version>1.0.0</version> <!-- {x-version-update;com.azure.spring:spring-cloud-azure-sample-stream-binder-servicebus-queue;dependency} -->
+  <version>1.0.0</version> <!-- {com.azure.spring:spring-cloud-azure-sample-stream-binder-servicebus-queue;dependency} -->
```
Comment (yiliuTo) [inline servicebus/spring-cloud-azure-stream-binder-servicebus/servicebus-queue-binder/pom.xml:15]: same here

**Review by yiliuTo — CHANGES_REQUESTED** [review]

**Review by yiliuTo — APPROVED** [review]
LGTM

**Pull request merged**